lib/utils.ts

```diff
@@ -42,12 +42,21 @@ export class Utils {
   }
 
   /**
-   * Returns the web2 object.
+   * Returns the web3 object.
    * When called for the first time, web3 is initialized from the Arc.js configuration.
-   * Throws an exception when web3 cannot be initialized or there is no default client
+   * Throws an exception when web3 cannot be initialized.
+   * @param {boolean} forceReload true to reload/retry web3
    */
-  public static getWeb3(): any {
-    if (Utils.web3) {
+  public static getWeb3(forceReload: boolean = false): any {
+
+    if (forceReload) {
+      delete Utils.web3;
+      Utils.alreadyTriedAndFailed = false;
+      if (typeof window !== "undefined") {
+        delete window.web3;
```

Comment:
Hmm. Won’t this eliminate access to meta mask version of web3? I think we can reload web3 a different way. Maybe by calling get accounts?

Reply:
We set window.web3 below.

Reply:
Having said  that, I do question how it would affect the following ine:

`if (typeof web3 !== "undefined")`

Reply:
I'm not clear how "calling get accounts" would work.  Can you describe?